common.lic

```diff
@@ -764,6 +764,10 @@ module DRC
     )
   end
 
+  def bold(text)
+    result = "<pushBold/>#{text}<popBold/>"
```

Comment:
🧹  The `result = ` assignment isn't necessary. Doesn't affect the outcome of the method nor a merge blocker.
```suggestion
    "<pushBold/>#{text}<popBold/>"
```